javalib/src/main/scala/java/time/Duration.scala

```diff
@@ -152,11 +152,21 @@ final class Duration private (seconds: Long, nanos: Int)
 
   def abs(): Duration = if (isNegative()) negated() else this
 
-  def addTo(temporal: Temporal): Temporal =
-    temporal.plus(normalizedSeconds, SECONDS).plus(normalizedNanos, NANOS)
+  def addTo(temporal: Temporal): Temporal = {
+    val t1 =
+      if (seconds == 0) temporal
+      else temporal.plus(seconds, SECONDS)
```

Comment:
Changed `normalizedSeconds` to `seconds` because this is what the observed behvaiour in JDK 8 suggests. In particular `ofNanos(-1).addTo(LocalDateTime.MIN.plusNanos(1))` throws an exception when it tries to add one second to the given `LocalDateTime`.